app/webpacker/css/admin_v3/components/buttons.scss

```diff
@@ -115,3 +115,19 @@ button:not(.plain):not(.trix-button),
 a.button {
   text-decoration: none;
 }
+
+.button.disruptive::before {
+  content: "";
+  display: inline-block;
+  width: 14px;
+  height: 14px;
+  margin-right: 3px;
+  position: relative;
+  top: 2px;
+
+  background-color: $color-5;
+  -webkit-mask: url("../images/admin_v3/icn-clear.svg") no-repeat center center;
+  mask: url("../images/admin_v3/icn-clear.svg") no-repeat center center;
+  -webkit-mask-size: cover;
+  mask-size: cover;
```

Comment:
Clever trick! I was curious to learn more, and found this method which looks like an even neater solution: https://stackoverflow.com/a/65147574/421243

It says that if we edit the .svg file to use the keyword `currentColor`, it will inherit from the css `colour:` property. Would you be willing to give it a try?

Reply:
Oh, and how about this icon that's already available in FontAwesome? https://fontawesome.com/icons/circle-xmark?f=classic&s=solid&pc=%23c85136
I think it's note quite the same, but might be worth asking Mario.

Reply:
Nice find thanks (I searched, but without success)! Waaayyyy simpler ;)